lib/miq_automation_engine/engine/miq_ae_service_model_base.rb

```diff
@@ -109,12 +109,12 @@ def self.wrap_results(results)
         if results.nil?
           ret = nil
         elsif results.kind_of?(Array)
-          ret = drb_return(results.collect { |r| wrap_results(r) })
+          ret = results.collect { |r| wrap_results(r) }
         elsif results.kind_of?(ActiveRecord::Relation)
-          ret = drb_return(results.collect { |r| wrap_results(r) })
+          ret = results.collect { |r| wrap_results(r) }
         elsif results.kind_of?(ActiveRecord::Base)
```

Comment:
@Fryguy @gmcculloug 
The wrap_results also converts ActiveRecord Objects into Service Model equivalents, by removing the wrap_results we loose that functionality. Leads to couple of hundreds of spec errors by removing this. Can we do those changes in a different PR.